src/new-components/toast/toast.module.css

```diff
@@ -0,0 +1,8 @@
+.container {
+    pointer-events: all;
```

Comment:
It took me a while to find out why this is needed, which turns out it's because it's disabled in the stacked notification element. 

I think you did intend for this to be controlled by the Notification component, as it has this same rule but unused there - maybe it is better to move this off of Toast and let Notification manage pointer events.

Reply:
Makes sense, will do it.